### PR TITLE
Experiment/mobilenetv3

### PR DESCRIPTION
**Pull request by asbharath**
- Added MobileNetV3 small and large for CIFAR version
- fixed depthwise_regularizer for MobileNetV3

### CODE REVIEW

File: src/models/generic_model.py [hunk 1]
```diff
@@ -92,14 +92,14 @@ def __init__(self, framework: str, conversion_params, factor=1, input_shape=(224
       self.model()
 
     if self.output_layer_before_up2tf:
-      self.x = self.layers().Dense(self.label_dim, use_bias=True, name='Logits', kernel_regularizer=self.weight_regularizer)(self.x)
+      self.x = self.layers().Dense(self.label_dim, use_bias=True, name='up2tf_after_dense_Logits', kernel_regularizer=self.weight_regularizer)(self.x)
```
Comment (MRLoghmani) [inline src/models/generic_model.py]: Minor comment:
I think there is no need to change the name here and in line 102. These two layers are never created together, so there is no issue with them having the same name. In addition, 'up2tf_after_dense_Logits' and 'up2tf_before_dense_Logits' could be confusing in the cases where there is no Upstride2TF layer involved.
If this somehow creates a conflict for Keras, then I would rather go for 'logits_before_up2tf' (line 95) and 'logits_after_up2tf' (line 102) to emphasize that the layers represent the logits, not up2tf.

**Review by MRLoghmani — APPROVED** [review]

File: src/models/generic_model.py (hunk 1 above)
Comment (asbharath) [reply src/models/generic_model.py]: updated accordingly.
